proof/refine/ARM/CNodeInv_R.thy

```diff
@@ -5241,6 +5241,11 @@ lemma invalid_Thread_CNode:
   apply (clarsimp simp: obj_at'_def projectKOs)
   done
 
+(* FIXME MOVE *)
+lemma all_Not_False[simp]:
+  "All Not = False"
+  by blast
+
```

Comment:
Have you tried if moving this to somewhere in Lib breaks stuff? It looks like a generally useful thing we should always have in scope.

Reply:
I'm going to do that now. Since I can't access pcr I'll use the PR check to see if/where it fails.

Reply:
This causes a bunch of failure, might want to do it separately.